lib/src/handler/w3c/element.dart

```diff
@@ -0,0 +1,169 @@
+import 'dart:math';
+
+import 'package:webdriver/src/common/request.dart';
+import 'package:webdriver/src/common/webdriver_handler.dart';
+
+import 'utils.dart';
+
+class W3cElementHandler extends ElementHandler {
+  @override
+  WebDriverRequest buildClickRequest(String elementId) {
+    return new WebDriverRequest.postRequest('${elementPrefix(elementId)}click');
+  }
+
+  @override
+  void parseClickResponse(WebDriverResponse response) {
+    parseW3cResponse(response);
+  }
+
+  @override
+  WebDriverRequest buildSendKeysRequest(String elementId, String keysToSend) {
+    return new WebDriverRequest.postRequest(
+        '${elementPrefix(elementId)}value', {
+      'text': keysToSend, // What geckodriver really wants.
+      'value': keysToSend // Actual W3C spec.
+    });
+  }
+
+  @override
+  void parseSendKeysResponse(WebDriverResponse response) {
+    parseW3cResponse(response);
+  }
+
+  @override
+  WebDriverRequest buildClearRequest(String elementId) {
```

Comment:
Use '=> new ...' for consistency with other parts of  your code.

Reply:
Thanks! Will do later in a cleanup CL as this is not blockingly bad.